filebeat/harvester/log.go

```diff
@@ -255,3 +226,21 @@ func (h *Harvester) updateOffset(increment int64) {
 func (h *Harvester) UpdateState() {
 	h.sendEvent(h.createEvent())
 }
+
+func (h *Harvester) close() {
+	// On completion, push offset so we can continue where we left off if we relaunch on the same file
+	if h.Stat != nil {
+		h.Stat.Offset <- h.getOffset()
```

Comment:
how about giving this a name? Something like: `h.Stat.Finalized(h.offset)` or `h.Stat.Over(...)`. `Done`, `Finishing`, ...